libs/langchain/langchain/text_splitter.py

```diff
@@ -762,6 +763,32 @@ def get_separators_for_language(language: Language) -> List[str]:
                 " ",
                 "",
             ]
+        elif language == Language.KOTLIN:
+            return [
```

Comment:
I don't know any kotlin -- should any other keywords be included?

https://kotlinlang.org/docs/keyword-reference.html